tests/integration.rs

```diff
@@ -1,18 +1,41 @@
 mod utils;
 
 use anyhow::Result;
-use utils::{get_redis_connection, prepare_dirs, start_redis_server_with_module};
+use redis::RedisError;
+use utils::{get_redis_connection, start_redis_server_with_module};
 
 #[test]
-#[ignore] // test will be run explicitly by a script
 fn test_hello() -> Result<()> {
-    prepare_dirs()?;
-
-    let _guards = vec![start_redis_server_with_module("hello")?];
-    let mut con = get_redis_connection()?;
+    let _guards = vec![start_redis_server_with_module("hello", 6479)?];
```

Comment:
Instead of hardcoding the port which is error-prone, we should use a function that allocates ports (e.g. a simple static counter that increments).

But why is this needed? Assuming tests don't run in parallel, we can just reuse the same port every time, can't we?

Reply:
When running the tests, add `-- --test-threads 1` to the `cargo test` command line to run the tests sequentially. That's why I didn't support multiple ports originally.
Do you think it's worth it to run the tests in parallel? Might make it harder to debug, and as long as they're fast it's not a problem. This may be worth discussing as a matter of principle -- what do other modules do with integration tests, run them sequentially or simultaneously?

Reply:
I guess we should support parallel testing just to be on the safe side, but let's do it in separate PR